Decisions.TruCap/TruCapRest.cs

```diff
@@ -14,22 +15,21 @@ public static async Task<HttpResponseMessage> TruCapGet(string url, TruCapAuthen
         
         try
         {
-            HttpResponseMessage response = await client.SendAsync(request);
+            HttpResponseMessage response = client.Send(request);
+            response.EnsureSuccessStatusCode();
+
+            Task<string> resultTask = response.Content.ReadAsStringAsync();
+            resultTask.Wait();
 
-            return response;
+            return resultTask;
         }
         catch (Exception ex)
         {
-            if (ex.Message.Contains("timed out"))
-            {
-                throw new Exception("TruCap+ took too long to respond and has timed out.", ex);
-            }
-
-            throw;
+            throw new BusinessRuleException("The request to TruCap+ was unsuccessful.", ex);
```

Comment:
We can remove the try/catch here since we're catching it everywhere we're using it.